README.md

```diff
@@ -18,9 +18,10 @@ Up to you! Design API contracts how you desire - as long as the UI team can use
 
 ## Requirements
 ### Functional
+* I as a user can create a point
+* I as a user can retrieve list of points
```

Comment:
I think this is a key requirement and we should include it (even though I see the point not to mention it and expect that people will implement it anyways)